src/test/java/com/zendesk/maxwell/schema/ddl/DDLParserTest.java

```diff
@@ -200,7 +200,8 @@ public void testParsingSomeAlters() {
 			"create table `shard1.foo` ( `id.foo` int ) collate = `utf8_bin`",
 			"ALTER TABLE .`users` CHANGE COLUMN `password` `password` VARCHAR(60) CHARACTER SET 'utf8' COLLATE 'utf8_bin' NULL DEFAULT NULL COMMENT 'Length 60 for Bcrypt'",
 			"create table `shard1.foo` ( `id.foo` int ) collate = `utf8_bin`",
-			"create table if not exists audit_payer_bank_details (event_time TIMESTAMP default CURRENT_TIMESTAMP())"
+			"create table if not exists audit_payer_bank_details (event_time TIMESTAMP default CURRENT_TIMESTAMP())",
+			"ALTER TABLE foo RENAME INDEX index_quote_request_follow_on_data_on_model_name TO index_quote_request_follow_on_data_on_model_class_name"
```

Comment:
oh so this is possible b/c it is just parsing testing, not being executed against mysql 5.5 and 5.6